src/arch/riscv/isa.hh

```diff
@@ -70,7 +70,7 @@ enum FPUStatus
 class ISA : public BaseISA
 {
   protected:
-    RiscvType rv_type;
```

Comment:
It has the public `rvType` for public function. Perhaps it should be `_rvType`?

Reply:
+1 I think this could cause a build issue?

Reply:
I changed all occurrences to _rvType and just changing that line was also causing errors. The errors are in isa.cc is it ok to change rv_type to _rvType there as well?